lib/mapper/product_mapper.rb

```diff
@@ -6,6 +6,8 @@ def self.map_payload(product, additional_data = {}, track_inventory = true, defa
                 name = product['name']
                 isDigital = product['isDigital'].to_s == 'true'
 
+                track_inventory = self.get_availability(product) == 'preorder' ? false : true
```

Comment:
@Jhalter5Stones This doesn't work.

The `track_inventory` value passed into this function will be `false` if the agent's `track_inventory` _option_ is set to `false`.

That should never be overwritten.